### ISSUE TITLE
simple deep learning solver for infinite horizon problem

### ISSUE THREAD

**Issue by sbenthall** [scikit-agent/scikit-agent#23]
Building on #22 -- the estimated lifetime reward loss function -- construct a simple solver for a Block that optimizes a neural network, representing the decision rule -- with respect to this loss function.
